README.md

```diff
@@ -127,10 +127,10 @@ For example, in the following example, `new D` would result in an object whose `
 
 ```js
 class C {
-  y = 1;
```

Comment:
Can we use `y` in this example, and change the mistaken reference to `x` above? This way, names don't overlap between examples.

Reply:
Alright